### DISCUSSION TITLE
Which CSL tests are missing?

### DISCUSSION THREAD

**Discussion by denismaier** [andras-simonyi/citeproc-el#40] [Q&A] [answered]
According to the readme this library currently covers 70% of the CSL test suite. What is still missing?

**Comment by andras-simonyi** [comment, marked as answer]
You can see all of the currently failing tests on the list [expected_fails,txt](https://github.com/andras-simonyi/citeproc-el/blob/master/test/expected_fails.lst). IIRC a large number of them is actually not about CSL proper, and many of them are irrelevant, although some covered functionality of course would surely be nice to have.
